wiki/Tournaments/4DM/2/en.md

```diff
@@ -59,12 +59,69 @@ The 4 Digit osu!mania World Cup 2 is run by various community members.
 - [Map suggestion form](https://forms.gle/BjLjtUSAVDLhhQ368)
 - **[Statistics sheet](https://docs.google.com/spreadsheets/d/1cT1BLBuG3XGyDvPWFTGJO_TOpRkdKQaH6shstbJcauc/edit?usp=sharing)**
 
+---
```

Comment:
these are for splitting content within a section, they dont belong at the end

Reply:
So just delete this one or others too?

Reply:
all of em